src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -1062,6 +1099,32 @@ object Desugar {
               }
             } yield in.Seqn(Vector(dPre, exprAss, clauseBody))(src)
 
+          case n@PContinue(label) =>
+            label match {
+              case None => unit(in.Continue(nm.fetchForId(n, 0, info))(src))
+              case Some(l) => unit(in.Continue(nm.fetchForId(n, info.enclosingLabeledLoopOrder(l, n), info))(src))
+            }
+
+          case n@PBreak(label) =>
+            label match {
+              case None =>
+                info.enclosingLoopNode(n) match{
+                  case None => violation("break should be enclosed in a loop")
+                  case Some(loop) =>
+                    for {
+                      (dInvPre, dInv) <- prelude(sequence(loop.spec.invariants map assertionD(ctx)))
```

Comment:
What is the purpose of this line here? In particular, `dInvPre` and `dInv` don't appear to be used